typesystem/schemas.py

```diff
@@ -4,11 +4,16 @@
 
 from typesystem.base import ValidationError, ValidationResult
 from typesystem.fields import Field, Object
+from typesystem.namespaces import SchemaNamespace, set_namespace
 
 
 class SchemaMetaclass(ABCMeta):
     def __new__(
-        cls: type, name: str, bases: typing.Sequence[type], attrs: dict
+        cls: type,
+        name: str,
+        bases: typing.Sequence[type],
```

Comment:
not 100% sure, but this should be `typing.Sequence[typing.Type]` i think

Reply:
I think it’s right as it stands. Type would also work, but makes more sense if we wanted specificity. E.g. can use `list` or `typing.List` for an annotation, but you’d usually only use the later for list-of-x

Reply:
ah thanks for explaining. i am still new to `typing`.